requirements.txt

```diff
@@ -0,0 +1,2 @@
+transformers==2.8.0
+torch==1.5.0
```

Comment:
I think we can leave the requirements.txt for now. We'll create a separate PR for that + other pre-requisite/dependency/setup.

Reply:
Sure. Will fix this